src/isomorphic/classic/types/ReactPropTypes.js

```diff
@@ -114,6 +115,11 @@ function createChainableTypeChecker(validate) {
     componentName = componentName || ANONYMOUS;
     propFullName = propFullName || propName;
     if (props[propName] == null) {
+      if (__DEV__) {
+        ReactInstrumentation.debugTool.onMissingProperty(componentName,
```

Comment:
`onCreateChainableTypeChecker` doesn’t seem like a good event name here to me. “Creating” type checker happens above this function. When this function runs, we are *checking* prop types, not *creating* a prop type checker. (`checkType` lives inside of `createChainableTypeChecker`).